update_compilers/install_python_compilers.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+. ${SCRIPT_DIR}/common.inc
+
+get_python() {
+    local VER=$1
+    local DIR=Python-$VER
+
+    if [[ ! -d ${DIR} ]]; then
```

Comment:
This is going to download the python source tree into `/opt/compiler-explorer/Python-{version}`, build it there, and then leave it - even though the final build artifacts will be put into `/opt/compiler-explorer/python`.

We don't need the source and build tree to stick around afterwards, and our disk footprint is enormous as is, so lets not do this.

Reply:
Good point, I'll make it rm the build dir after it's done

Reply:
Ideally can you build it from a temporary directory? The `/opt/compiler-explorer/` is an NFS directory so accessing it is terribly slow (unfortunately).

Reply:
It could be built from a temp directory, is there a best way to do that? I'm not super familiar with bash scripts that use temporaries.

Reply:
No worries, actually I'll take a look at merging and fixing up in the way Austin suggests :) Thanks!